SnapMD.VirtualCare.ApiModels/Scheduling/AppointmentStatusCode.cs

```diff
@@ -32,6 +32,16 @@ public enum AppointmentStatusCode
         /// <summary>
         /// The patient was transferred to a different doctor.
         /// </summary>
-        Transferred
+        Transferred,
```

Comment:
@jay-lee00 @simplecode80 @Number1MSFanboy @dacaballero @nonsense66 Per our earlier discussion about the difference between Appointment Blocks and video encounters, statuses like "dropped" are not appropriate properties of an Appointment object. Per the HL7 guidelines located at https://www.hl7.org/fhir/appointment.html (READ THIS), it would be beneficial for us to replace the "InProgress" status with "Fuflilled".  After an appointment is fulfilled through the generation of a Consultation/Encounter, there is no further reason to track the appointment. Further consultation life cycle events should be attached to the Consultation/Encounter object, not the Appointment. Thank you.